src/main/java/frc/team4909/robot/subsystems/climber/ClimberSubsystem.java

```diff
@@ -0,0 +1,58 @@
+package frc.team4909.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import frc.team4909.robot.RobotMap;
+import frc.team4909.robot.RobotConstants;
+import frc.team4909.robot.subsystems.drivetrain.commands.Drive;
+import frc.team4909.robot.subsystems.climber.commands.ExtendStilts;
+import frc.team4909.robot.subsystems.climber.commands.RetractStilts;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import frc.team4909.robot.Robot;
+import frc.team4909.robot.operator.controllers.BionicF310;
+
+public class ClimberSubsystem extends Subsystem {
+    WPI_TalonSRX climberSRX;
+    WPI_VictorSPX climberSPX;
+
+    public ClimberSubsystem() {
+        climberSRX = new WPI_TalonSRX(RobotMap.climberSRXID); // Climber lift
+        climberSPX = new WPI_VictorSPX(RobotMap.climberSPXID); // CLimber drive
+
+        climberSRX.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative);
+
+        /* PID yet to be calibrated */
```

Comment:
personal thing, but I like to prefix all of my remaining work comments with `TODO: ` so that it's easy to check if you finished everything by doing the file search across the workspace... either way, I'd make sure this gets tuned tomorrow(errr i guess today now) if possible